src/utils/getWeightDivID.js

```diff
@@ -0,0 +1,41 @@
+const csv = require("csv-parser");
+const fs = require("fs");
+const path = require("path");
+
+const getWeightDivID = async (sex, weight, age) => {
+  // Dynamically find the path to the age_div.csv file (workaround for Jest testing environment)
+  let weightDivPath = __dirname;
+  while (!weightDivPath.endsWith("src")) {
+    weightDivPath = path.resolve(weightDivPath, "..");
+  }
+  weightDivPath = path.resolve(
+    weightDivPath,
+    "app",
+    "sql",
+    "database",
+    "weight_div.csv",
+  );
+
+  const isYouth = age < 14 ? 1 : 0; // Determine if participant is in youth division
```

Comment:
could you do this instead?
```suggestion
  const isYouth = age < 14;
```

Reply:
Just tried it out and sadly I can't bc itll mess it up